index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="<link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="header">
+    <h1 class="Mylogo"><a href="#">My Logo</a></h1>
+    <nav>
+      <a href="#"><img src="images/Union.png" alt="Union"></a>
+    </nav>
+  </header>
+  <main>
+    <section class="headline">  
+      <h2>I’m Henary
+        Glad to see you!</h2>
```

Comment:
- [ ] Kindly use `<br />` tag to make the title 2/3 lines to match the given design.
```suggestion
            <h1>
              I am Henry <br />
              Great to meet you!!
            </h1>
```

Reply:
Thank you for finding out my mistakes. I will makes changes as required.